posthog/session_recordings/queries/session_replay_events.py

```diff
@@ -489,6 +489,35 @@ def get_sessions_from_distinct_id_query(
         )
         return query
 
+    @staticmethod
+    def get_sessions_from_team_id_query(
+        format: Optional[str] = None,
+    ):
+        """
+        Helper function to build a query for listing all session IDs for a given team ID
+        """
+        query = """
+                SELECT
+                    session_id,
+                    min(min_first_timestamp) as start_time,
+                    max(retention_period_days) as retention_period_days,
+                    dateTrunc('DAY', start_time) + toIntervalDay(coalesce(retention_period_days, %(ttl_days)s)) as expiry_time
+                FROM
+                    session_replay_events
+                PREWHERE
+                    team_id = %(team_id)s
+                    AND min_first_timestamp <= %(python_now)s
```

Comment:
_maybe_ we should page backwards by time  
in 90 day blocks  
  
so for larger teams this query doesn't get unwieldy to run  
  
but that's easy enough to add if it turns out to be useful